mod/consensus/pkg/cometbft/service/service.go

```diff
@@ -58,11 +58,13 @@ const (
 
 const InitialAppVersion uint64 = 0
 
-type Service struct {
+type Service[
+	LoggerT log.AdvancedLogger[LoggerT],
```

Comment:
have to add generic here for now because wrapcometlog/wrapsdklog requires `logger.With()` method